servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -133,7 +133,11 @@
    */
   @SuppressFBWarnings("URF_UNREAD_FIELD")
   private Future<?> mTtlCheckerService;
-
+  /**
+   * Creates a new list of {@link TtlBucket}s.
```

Comment:
this is not a method ... so the JavaDoc should not have `@return` and it should not describe an action, it should describe the purpose of the field